### DISCUSSION TITLE
Unlocking a page raises exception

### DISCUSSION THREAD

**Discussion by jrief** [django-cms/djangocms-versioning#368] [Q&A] [answered]
In `djangocms_versioning/admin.py:1276` (unlock_view) the function `notify_version_author_version_unlocked` is called which itself attempts to send an email to the user owning the lock.

However, if the project configuration does not  configure an email backend, then this function can't connect and raises

`OSError: [Errno 101] Network is unreachable`. 

In my opinion, such a network issue shall be logged but not cause an internal server error.

**Comment by fsbraun** [comment]
And, if no email backend is configured, it should not even try? (I.e. logs are not bloated with failed notifications if none are configured).

**Comment by jrief** [comment, marked as answer]
In projects settings, set `EMAIL_NOTIFICATIONS_FAIL_SILENTLY = True` to prevent this.

**Reply by fsbraun** [reply]
Still, would it not make sense if no email backend is configured to fail silently? Now, you would either have to add the above setting or configure and email backend. Can't we include one pair of batteries?
